### PR TITLE
Editionalised navigation on AMP

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?

This adds the ability to have an editionalised menu on amp using `amp-sidebar` and `amp-list`.

Route has to be added to the platform repo as well. Stay tuned for that PR!

*Please note* the last two commits are refactoring the liveblog css. This is because the added css in this PR was enough to push us over out 50000 byte css limit for AMP pages. Refactoring the liveblog css to remove duplication fixed this issue.

## What is the value of this and can you measure success?
- More parity between AMP and www
- More selection for navigation for users
- Beautiful design

## Does this affect other platforms - Amp, Apps, etc?
This should only affect AMP

## Screenshots
Before:
image

After:
image

and open:
image

image

@guardian/dotcom-platform

### CODE REVIEW

File: onward/app/controllers/NavigationController.scala [hunk 1]
```diff
@@ -1,37 +1,71 @@
...
 
-      RevalidatableResult.Ok(arr(Edition.all.map { edition =>
-        obj(edition.id -> arr(edition.navigation.map(item => NavItemAndEdition(item, edition))))
+      RevalidatableResult.Ok(Json.arr(Edition.all.map { edition =>
+        Json.obj(edition.id -> Json.arr(edition.navigation.map(item => NavItemAndEdition(item, edition))))
       }))
     }
   }
+
+//  This is to editionlise the menu on AMP
```
Comment (TBonnin) [inline onward/app/controllers/NavigationController.scala]: typo `editionlise`

File: onward/app/controllers/NavigationController.scala [hunk 2]
```diff
@@ -1,37 +1,71 @@
...
+              Json.obj(
+                "title" -> section.title,
+                "url" -> LinkTo(section.url)
+              )
+            )
+          )
+        ))
+      )
+    }
+  }
```
Comment (TBonnin) [inline onward/app/controllers/NavigationController.scala]: Would it be possible to abstract the `Section` to `json` bit? Maybe using implicit `Writes` (see example at the beginning of this file

**Comment by GHaberis** [comment]
@NataliaLKB the CSS & HTML are looking tidy to me! 👍

File: onward/app/controllers/NavigationController.scala (hunk 2 above)
Comment (NataliaLKB) [inline onward/app/controllers/NavigationController.scala]: I have refactored things to use the implicit Write converters. I think it is much more readable now. What do you think?

**Review by TBonnin — COMMENTED** [review]
Scala code looks good to me 👍

**Comment by NataliaLKB** [comment]
Thanks @TBonnin @GHaberis! I am at a bit of a loss why the css compilation is failing in teamcity. Everything works fine locally. I will keep this PR open while I investigate this.

**Pull request merged**